container-storage-setup.conf

```diff
@@ -1,7 +1,7 @@
 # Specify storage driver one wants to use with container runtimes.
-# Default is devicemapper.
-# Other possible options are overlay, overlay2 and "". Empty string means do
-# not do any storage setup.
+# The default is `devicemapper`.
+# Other possible options are "auto", "overlay" as
+# well as the empty string "".  For more information, see `man container-storage-setup`.
```

Comment:
overlay and overlay2 are two different drivers in docker. So not sure why should we remove overlay2 from list.